### PR TITLE
Various fcct usage updates

### PR DESCRIPTION
**Pull request by bgilbert**
The last commit depends on https://github.com/coreos/fcct/pull/72 and the second-to-last one should probably not happen without https://github.com/coreos/fcct/pull/70.

### CODE REVIEW

**Comment by lucab** [comment]
It looks like the first two commits make sense on their own, and would be good to merge quickly to prepare for the rest of changes (esp. the single-dash breaking).

Should we maybe split this PR in order to land the first part right now? Other than that, LGTM.

File: modules/ROOT/pages/using-fcct.adoc [hunk 1]
```diff
@@ -15,24 +15,22 @@ You can run `fcct` as a container with docker or podman or download it as a stan
 
 This example uses podman, but you can use docker in a similar manner.
 
-In this example, substitute `v0.2.0` with the desired version, as the `latest` tag corresponds with `master` and not with the latest release.
```
Comment (lucab) [inline modules/ROOT/pages/using-fcct.adoc]: Nit: maybe add an explicit note that `release` always tracks latest release, and `latest` tracks git development?

**Comment by bgilbert** [comment]
Updated.  The three commits in this PR should be good to merge now, and the last commit has been moved to https://github.com/coreos/fedora-coreos-docs/pull/38.

File: modules/ROOT/pages/producing-ign.adoc [hunk 2]
```diff
@@ -51,7 +51,7 @@ An easy way to use `fcct` is to run it in a container with `podman`:
 +
```
Comment (jlebon) [inline modules/ROOT/pages/producing-ign.adoc]: Optional: a link back to https://github.com/coreos/fcct/pull/70 in this commit's message for posterity would be good.

(Grrr... on GitHub for not allowing commenting on a commit message. I'm talking about 8f4c3f200768662a61d0ef3481d1438d780fe25c here.)

**Comment by bgilbert** [comment]
Updated!

**Pull request merged**